tox.ini

```diff
@@ -0,0 +1,144 @@
+# Differences from the starbase tox.ini:
+# * Uses requirements-devel.txt
+# * No library code for pyright type checking
+# * Documentation envs removed
+# * Pytest uses wheel package
+# * Legacy tests instead af integration tests
+# * Additional test environment "noreq" runs all the tests without requirements.txt
+# * lint-shellcheck includes spread-shellcheck.py
+# * Additional ruff configuration for snapcraft_legacy
+# * Include isort
```

Comment:
Curious - isort is not in starbase because ruff implements it?

Reply:
Yes. Ruff's isort rules are... *slightly* different from our isort configuration here (stricter, so running ruff with isort enabled wants to make some changes in `snapcraft_legacy` that isort doesn't). Ideally I'd like to phase out isort and use ruff's isort rules, but that's its own change.